frontend/src/index.css

```diff
@@ -91,18 +91,20 @@ input[type="number"] {
   margin: 0 2.25rem 0 3.75rem;
 }
 
-.save-button {
+.basic-button {
   background-color: hsl(76deg 55% 53%);
   color: hsl(0deg 0% 15%);
   border: none;
   border-radius: 4px;
-  padding: 1rem 1.5rem;
   box-shadow: 0 3px 6px hsl(0deg 0 15%);
   box-shadow: 0 2px 4px hsl(0deg 0 12%);
   transition-duration: 0.2s;
+  margin-left: 1rem;
+}
+
+.save-button {
   margin: 2rem;
-  position: absolute;
-  right: 0;
+  padding: 1rem 1.5rem;
 }
 
 .save-button:hover {
```

Comment:
I realized that the hover and disabled effects only work on the save button, I missed that before. I think it would be nice if they were there for all buttons, or what do you think? In that case you could just change this to 
`.basic-button:hover` 
and further down
`.basic-button:disabled`

Reply:
ah yes, that was the goal, thanks